### DISCUSSION TITLE
Sidebar sites with different containers.

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#868] [Ideas]
### What feature would you like to see?

If it's possible to add the multi account features on the sidebar websites so I can open different websites in different containers.

### What platform are you seeing the problem on?

Windows

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/265_

**Comment by theherk** [comment]
Would be excellent for this to be implemented like [Arc's Air Traffic Control](https://resources.arc.net/hc/en-us/articles/22932014625431-Air-Traffic-Control-Automate-Your-Link-Routing). Basically, that opens each link in a given Space, and each Space has its own profile. So effectively they have completely different credentials and settings for each. This is extremely useful, and frankly one of Arc's best features. To do the same with multi-account containers would be awesome.
